test/controllers/update/overseas.entity.presenter.controller.spec.ts

```diff
@@ -185,4 +198,98 @@ describe("OVERSEAS ENTITY PRESENTER controller", () => {
       expect(resp.text).not.toContain(ErrorMessages.EMAIL_INVALID_FORMAT);
     });
   });
+
+  describe("POST tests on a remove journey", () => {
+    test(`redirect to ${UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_PAGE} page after a successful post from overseas entity presenter page`, async () => {
+      const resp = await request(app).post(`${OVERSEAS_ENTITY_PRESENTER_URL}${JOURNEY_REMOVE_QUERY_PARAM}`).send(PRESENTER_OBJECT_MOCK);
+
+      expect(resp.status).toEqual(302);
+      expect(resp.text).toContain(`${FOUND_REDIRECT_TO} ${UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL}`);
```

Comment:
We could also check that the remove banner is is in the response for all theses tests:
` expect(resp.text).toContain(config.REMOVE_SERVICE_NAME);`

Reply:
i don't think we can check for that banner if we get a 302, we can only check it in the GET tests and the POST error page tests, so check the banner in those

Reply:
added the test to Get tests and POST tests